docs/content/docs/how-to-guides/2_configuringEnbuild.md

```diff
@@ -33,6 +33,7 @@ ENBUILD supports GitHub, GitLab VCS as of now.
 These are the Version Control System where ENBUILD creates repositories when you deploy catalog item
 
 ### GITHUB
+Navigate to Admin Settings and select GITHUB under, Set and configure your VCS account. Once selected click the configure button at the top right of the GITHUB box you have selected. 
```

Comment:
By default is Github is disabled, To enable the Github first you need to 
- Navigate to Admin Settings and select GITHUB under, Set and configure your VCS account. Once selected click the configure button at the top right of the GITHUB box you have selected